akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -164,11 +164,20 @@ when using polymorphic types.
 
 ### ADT with trait and case object
 
-In Scala it's common to use a sealed trait and case objects to represent enums. If the values are case classes
+In Scala, it's common to use a sealed trait and case objects to represent enums. If the values are case classes
```

Comment:
This comma sounds more weird to me than the original sentence?

Reply:
I just followed the grammar suggestion from IntelliJ

![CleanShot 2020-09-01 at 12 53 05](https://user-images.githubusercontent.com/502982/91841175-2c202900-ec52-11ea-94cd-21ccfa51023e.png)

Reply:
the name of the grammar rule is: "Missing comma after an introductory phase"

Reply:
This is not an introductory phrase. Only a bit unusual sentence order: "It is common in Scala to ..."